packages/cli/src/normalizeConfig.js

```diff
@@ -38,23 +39,7 @@ export async function normalizeConfig(inConfig) {
       let newEleventyConfig = fileConfig.eleventy || {};
       let eleventyFunction = config.eleventyFunction;
       if (typeof fileConfig.eleventy === 'function') {
-        newEleventyConfig = fileConfig.eleventy({
-          addLiquidFilter: () => {},
-          addNunjucksFilter: () => {},
-          addHandlebarsHelper: () => {},
-          addJavaScriptFunction: () => {},
-          addFilter: () => {},
-          addLiquidShortcode: () => {},
-          addNunjucksShortcode: () => {},
-          addHandlebarsShortcode: () => {},
-          addShortcode: () => {},
-          addLiquidTag: () => {},
-          addNunjucksTag: () => {},
-          setTemplateFormats: () => {},
-          setQuietMode: () => {},
-          addTransform: () => {},
-          addLinter: () => {},
-        });
+        newEleventyConfig = fileConfig.eleventy(eleventyConfig);
```

Comment:
I'm not sure if that would have side effects 🤔

adding `addPlugin: () => {},` might be saver 🤔

Reply:
Ok, I will check all functions we could need and include at once

Reply:
I created an object with all empty functions that I guess could be needed

Reply:
nice 👍